Testscripts/Linux/TestRDMA_MultiVM.sh

```diff
@@ -169,6 +170,64 @@ function Run_IMB_MPI1() {
 	fi
 }
 
+function Run_OSU_P2P() {
+	total_attempts=$(seq 1 1 $imb_mpi1_tests_iterations)
+	if [[ $osu_p2p_tests == "all" ]]; then
+		extra_params=""
+	else
+		extra_params=$osu_p2p_tests
+	fi
+	for attempt in $total_attempts; do
+		LogMsg "OSU P2P test iteration $attempt for $mpi_type- Running."
+		case "$mpi_type" in
```

Comment:
If this benchmark test is only for MVAPICH, then use 'if-statement'.

Reply:
Currently it is for mvapich.. I will adding for other MPIs in subsequent PRs